src/components/Nav/Nav.js

```diff
@@ -0,0 +1,284 @@
+import { useEffect, useState } from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { BiSearchAlt } from 'react-icons/bi';
+import { BsCart2 } from 'react-icons/bs';
+import { IoIosArrowDown } from 'react-icons/io';
+
+export function Nav() {
+  const [click, setClick] = useState(null);
+  const [isStore, setIsStore] = useState(false);
+  const [position, setPosition] = useState(false);
+
+  useEffect(() => {
+    let lastScrollY = 0;
+    let result = true;
+    const updateScroll = () => {
+      const scrollY = window.scrollY;
+      const direction = scrollY > lastScrollY;
+      lastScrollY = scrollY;
+      if (direction === result) {
+        return;
+      }
+      result = direction;
+      return setPosition(direction);
+    };
+    window.addEventListener('scroll', updateScroll);
+    return () => {
+      document.removeEventListener('scroll', updateScroll);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (click === 'store') {
+      setIsStore(true);
+    }
+    if (click === 'community') {
+      setIsStore(false);
+    }
+  }, [click]);
+
+  const clickTab = terget => {
+    setClick(terget);
+  };
+
+  const subCategory = () => {
+    return isStore ? STORE_DATA : COMMUNITY_DATA;
+  };
+
+  return (
+    <>
+      <NavBar>
+        <NavWrapper>
+          <NavDiv>
+            <Link to="/">
+              <img alt="" src="/images/Nav/logo.png" style={{ width: 100 }} />
+            </Link>
+            <NavUl>
+              {NAV_DATA.map(item => (
+                <NavLi
+                  key={Math.random()}
+                  onClick={() => clickTab(item.category)}
+                >
+                  <NavLinkStyled
+                    className={({ isActive }) => (isActive ? 'active' : '')}
+                    to={item.link}
+                  >
+                    {item.title}
+                  </NavLinkStyled>
+                </NavLi>
+              ))}
+            </NavUl>
+            <SearchDiv>
+              <SearchIcon />
+              <input type="text" placeholder="통합검색" />
+            </SearchDiv>
+            <NavUl>
+              {MEMBER_DATA.map(item => (
+                <NavLi
+                  key={Math.random()}
+                  onClick={() => clickTab(item.category)}
+                >
+                  <NavLink to={item.link}>
+                    {!item.title ? <Cart /> : item.title}
```

Comment:
부정 조건을 조건으로 하는 것은 지양해주세요!
```suggestion
                    {item.title ? item.title : <Cart />}
```
추가적으로, 이렇게도 할 수 있을 것 같습니다.
```suggestion
                    {item.title || <Cart />}
```